build.sbt

```diff
@@ -463,8 +489,10 @@ lazy val docs = project
     watchSources += (ThisBuild / baseDirectory).value / "docs",
     Global / cancelable := true,
     MdocPlugin.autoImport.mdoc := (Compile / run).evaluated,
-    mdocJS := Some(jsdocs),
-    mdocJSLibraries := (jsdocs / Compile / fullOptJS / webpack).value,
+    mdocJS := Some(jswebsitedocs),
+    // mdocJSLibraries := (jsdocs / Compile / fullOptJS).outputFiles.map { path =>
```

Comment:
Both would be needed. The jsdocs project appears to be used quite extensively in testing - my recollection was that tampering with it would be a large undertaking, as it fulfils multiple roles in the project. 

The `jsdocswebsite` project is now used solely for the website.